packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -248,6 +248,7 @@ class SortedTable extends Component {
   static propTypes = {
     defaultColumn: PropTypes.number,
     defaultFilter: PropTypes.string,
+    displayFilter: PropTypes.bool,
```

Comment:
Please add a comment to say why we need it, we may find a better solution in the future and get rid of it.